src/jit/CMakeLists.txt

```diff
@@ -138,6 +142,7 @@ set( JIT_ARM64_SOURCES
   lowerarm64.cpp
   lsraarmarch.cpp
   lsraarm64.cpp
+  simd.cpp
```

Comment:
You also need to add simdcodegenarm64.cpp

Reply:
Right now I put all the simdcodegen in codegenarm64.  If it is preferred I can create a separate file.